test/corpus/types-pending.txt

```diff
@@ -0,0 +1,39 @@
+
+================================================================================
+Subclass with multiline params and unnamed after named param
```

Comment:
Isn't it a duplicate of "For-comp followed by string"?

Reply:
Good catch, thanks. I did a few passes and sometimes forgot which `-pending.txt` file had which examples.